config/checkstyle-config.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE module PUBLIC
     "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
```

Comment:
Shell we update/fix the 'Puppy Crawl' comment as well?

Reply:
done